### DISCUSSION TITLE
Allow viewers to silence alarms

### DISCUSSION THREAD

**Discussion by terrpan** [grafana/grafana#60867] [Alerting]
We want to use Grafana as a central GUI to manage alerts with Prometheus alert manager. But right now it does not look like viewers can silence alerts. The editor role is a bit to broad for us to assign to users.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82819. Feel free to continue the discussion around this there. Thank you!
